### DISCUSSION TITLE
Is macOS 13 Ventura supported?

### DISCUSSION THREAD

**Discussion by spacejam** [lima-vm/lima#1229] [Q&A] [answered]
Hello, I use Lima for my primary work environment, and would like to know if Lima is known to be functional on Ventura before making the jump. I'm extremely happy with Lima overall and it would not be worth updating if it is not known to work well on macOS 13 yet. FWIW I searched the readme and issues related to Ventura, and I see that there has been some macOS 13-related work happening, but I do not see information that leads me to feel confident in updating yet.

As a suggestion, some people may benefit from a "Known Stable macOS Version" section of the readme, to help others trying to make a similar decision in the future.

**Comment by AkihiroSuda** [comment, marked as answer]
Yes, the latest version (v0.13) should work on Ventura
